### PR TITLE
Document new -XX:[+|-]MergeCompilerOptions

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
Added a new topic for the -XX:[+|-]MergeCompilerOptions option and updated the related topics

Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

Closes https://github.com/eclipse-openj9/openj9-docs/issues/1015

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@dsouzai @hzongaro  - I have updated the documents. Could you please check and let me know if any further changes are required. 
Thanks!

File: docs/xxmergecompileroptions.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Explanation
+
+ By default, if you specify `-Xjit` option (or `-Xaot` option) multiple times, only its last option takes effect. Therefore, on adding an `-Xjit` option, the resultant behavior might not be as expected, if any instance of this option already exists. It is time-consuming to find the existing instances and try combining them manually so that the system behaves as required.
+
+ You can use the `XX:+MergeCompilerOptions` option to merge all the existing instances of the `-Xjit` options logically. To retain the default behavior, whereby only the last option takes effect, you can use the `XX:-MergeCompilerOptions` option.
+
+## Example
+
+No compiler options:
```
Comment (hzongaro) [inline docs/xxmergecompileroptions.md:46]: These "No compiler options:", "One JIT option:" headings might look better as items in a bulleted list or subheadings or something, just to make them stand out a bit more.

File: docs/xxmergecompileroptions.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
```
[the remark this answers is not in the document]
Comment (doveye) [reply docs/xxmergecompileroptions.md:2]: The copyright dates need to match existing topics, as described in the [contribution guidelines](https://github.com/eclipse-openj9/openj9-docs/blob/master/CONTRIBUTING.md#style-guidelines). The reason is that the dates are for the "work" as a whole (the "work" is the thing that the copyright office would register copyright for). You can think of a topic as a like chapter in a book - the copyright statement applies to the whole book not an individual chapter.

File: docs/xxmergecompileroptions.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+This option enables or disables the merging of multiple `-Xjit:` or `-Xaot:` options into a single `-Xjit` or `-Xaot` command.
+
+## Syntax
+
+        -XX:[+|-]MergeCompilerOptions
+
+| Setting                    | Effect  | Default                                                                              |
+|----------------------------|---------|:------------------------------------------------------------------------------------:|
+|`-XX:+MergeCompilerOptions` | Enable  |                                                                                      |
+|`-XX:-MergeCompilerOptions` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
```
[the remark this answers is not in the document]
Comment (hzongaro) [reply docs/xxmergecompileroptions.md:36]: I see - this is something that will work when it's published in eclipse.org

[the remark this answers is not in the document]
Comment (doveye) [reply docs/xxmergecompileroptions.md:36]: @hzongaro are you looking at the topic at https://eclipse-openj9.github.io/openj9-docs-staging/1027/xxmergecompileroptions/ or in github? The fontawesome string is converted during the doc build to an svg so you won't see the icon unless you're looking at a built topic.

[the remark this answers is not in the document]
Comment (doveye) [reply docs/xxmergecompileroptions.md:36]: Yes, and you can see it in the draft docs now at https://eclipse-openj9.github.io/openj9-docs-staging/1027/xxmergecompileroptions/ (when a PR is staged using the "jenkins doc stage" phrase, you can see the update by using a URL of this form with the number changed to match the PR).

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

File: docs/version0.36.md [hunk 4]
```diff
@@ -29,6 +29,7 @@ The following new features and notable changes since version 0.35.0 are included
 - [New binaries and changes to supported environments](#binaries-and-supported-environments)
 - [New `-XX:JITServerAOTmx` option added](#new-xxjitserveraotmx-option-added)
 - [Changes to the location of the default directory for the shared cache and snapshot](#changes-to-the-location-of-the-default-directory-for-the-shared-cache-and-snapshot)
+- [New `-XX:[+|-]MergeCompilerOptions` option added](#new-xx-mergecompileroptions-option-added)
```
[the remark this answers is not in the document]
Comment (Sreekala-Gopakumar) [reply docs/version0.36.md:32]: Please check in https://eclipse-openj9.github.io/openj9-docs-staging/1027/version0.36/. It is working when it is built. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
@hzongaro - I have made the changes. Please check and confirm. Thanks!

File: docs/version0.36.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hzongaro) [reply docs/version0.36.md:32]: Interesting.  I guess there must be differences in how different tools create the URL fragment from the heading.

**Review by hzongaro — APPROVED** [review]
Looks good!

**Comment by Sreekala-Gopakumar** [comment]
@dsouzai - Can these changes be merged? Thanks!

File: docs/xxmergecompileroptions.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+## Explanation
+
+ By default, if you specify `-Xjit` option (or `-Xaot` option) multiple times, only its last option takes effect. Therefore, on adding an `-Xjit` option, the resultant behavior might not be as expected, if any instance of this option already exists. It is time-consuming to find the existing instances and try combining them manually so that the system behaves as required.
+
+ You can use the `-XX:+MergeCompilerOptions` option to merge all the existing instances of the `-Xjit` options logically. To retain the default behavior, whereby only the last option takes effect, you can use the `-XX:-MergeCompilerOptions` option.
```
[the remark this answers is not in the document]
Comment (hzongaro) [reply docs/xxmergecompileroptions.md:42]: Yes, I agree.

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@dsouzai - Updated the document. Please merge the files if the changes are ok. Thanks!

**Review by dsouzai — APPROVED** [review]

**Pull request merged**